src/views/Dissolutions/ReviewConfirmDissolution.vue

```diff
@@ -156,8 +178,8 @@
         <label class="font-weight-bold pl-2">Affidavit</label>
       </header>
 
-      <div class="section-container" :class="{ 'invalid-section': isAffidavitInvalid }">
-        <section v-if="isAffidavitInvalid">
+      <div class="section-container" :class="{ 'invalid-section': !isAffidavitValid }">
+        <section v-if="!isAffidavitValid">
```

Comment:
I agree: "is valid" is easier to understand than "is invalid" 👍

Reply:
Definitely agree, good clean up!